python/sdk/docs/requirements_docs.txt

```diff
@@ -17,7 +17,7 @@ mlflow>=1.2.0,<=1.23.0
 google-cloud-storage>=1.19.0
 urllib3>=1.23
 PyPrind>=2.11.2
-google-auth>=1.11.0,<2.0dev
+google-auth>=2.16.2
```

Comment:
Same comment as https://github.com/caraml-dev/turing/pull/324/files#r1134052734

(Also about the `setup.py` file below.)

Reply:
Yep, I think it's redundant too. Just removed it! (In the `setup.py` file too)